index.js

```diff
@@ -164,6 +219,15 @@ function fastifyWebsocket (fastify, opts, next) {
         client.close()
       }
     }
+
+    // Close all the mocked socket used by injectWS
+    // This is needed to avoid test hanging
+    if (injectedWs.length) {
+      for (const ws of injectedWs) {
+        ws.terminate()
+      }
+    }
```

Comment:
This seems a bit sketchy to me -- it means that test websockets work differently than non-test websockets, so that if you write a test that uses `injectWS`, it will behave differently when you ship to production. Is there a way to make the injected WS use a much more similar path to the normal ones? I think it'd be ok to require each test that opens a websocket to close it at the end too.

Reply:
I didn't feel it was ok to close the websocket created inside the plugin manually.  If we agree that is ok to close it every time at the end I will happily remove this line.